package.json

```diff
@@ -17,8 +17,8 @@
   },
   "license": "Apache-2.0",
   "engines": {
-    "node": "4.x.x",
-    "npm": "3.x.x"
+    "node": "4.4.x",
+    "npm": "3.9.x"
```

Comment:
Not particularly, other than that it is what we have been using for a while now, and not updated lately.
Could we update that separately, since that requires us to update our builds on dcos and docs-enterprise repos